src/io/flutter/run/bazelTest/BazelTestConfigProducer.java

```diff
@@ -30,9 +31,10 @@ protected BazelTestConfigProducer() {
     super(FlutterBazelTestConfigurationType.getInstance());
   }
 
-  private static boolean isFlutterContext(@NotNull ConfigurationContext context) {
+  private static boolean isBazelFlutterContext(@NotNull ConfigurationContext context) {
     final PsiElement location = context.getPsiLocation();
-    return location != null;
+    final Workspace workspace = Workspace.load(context.getProject());
```

Comment:
DBC: I couldn't see how this worked initially, as I thought `Workspace` was something generic - perhaps IntelliJ related. In a future PR, we may want to rename to something like `BazelWorkspace` in order to better qualify the class.

Reply:
Noted.